distribution/test/itests/test-itests-common/pom.xml

```diff
@@ -176,6 +176,7 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.5.1</version>
```

Comment:
❔ Do you recall what the previous version was being used here?

Reply:
It was 3.5.1 for everything, but it was adding in an osgi.contraint that just wasn't needed and I couldn't figure out how to remove it. Setting it back to the older plugin got the tests to pass again and this is just a test bundle anyways.

Reply:
Is it worth a comment?

Reply:
Maybe? I mean, I don't care if someone tries to update it again in the future. If it doesn't work, you figure it out really quick. If it does work, then it should get updated. I'm a little afraid that if I leave a comment on there that no one will ever try to upgrade it ever again.

Reply:
I was more thinking of a comment saying what did not work so they would know how to test if it does work in the future.  If it is an obvious failure then maybe it is not necessary.  Though I bet someone tries to ask you in the future why you did this.  A comment might save them that time.